config/initializers/devise.rb

```diff
@@ -0,0 +1,299 @@
+# frozen_string_literal: true
+
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = '02e30cfac12f3cfe9c99c7047a65cf00e918db2f5664f37fa04382b52725a7bc5b17eb9ca9f145dd3b39ea8779c0030b2f8ce3cb4061d3cdeec1a0fbb2e54f7f'
```

Comment:
This needs to be uncommented for the README to work as-is. Either change the README instructions on how to setup the application or uncomment this line out.

Reply:
Hey Adnan,

Please correct me if I'm wrong, but I'm pretty sure this is being caused because there's no value for `secret_key_base` in `secrets.yml` unless you decrypt `application.yml` or replace the values in secrets.yml yourself. Did you try running it after you replaced all the keys in secrets.yml? The readme does say to do this, but I tried to make it more clear and pushed that change. 

Thanks,

Walker Spence

Reply:
Yeah, I didn't understand that those are the same. I'll merge the code with your README changes.